files/image_config/hostcfgd/hostcfgd

```diff
@@ -72,6 +78,8 @@ class AaaCfg(object):
                 self.auth['failthrough'] = is_true(data['failthrough'])
             if 'debug' in data:
                 self.debug = is_true(data['debug'])
+            if key == 'accounting':
```

Comment:
Tabbing does not look right. Should be at the same level as 'key == authentication"

Reply:
Thanks for the review. 
Indentation level has been corrected, please check again.
Thank you!